src/python/pants/backend/cc/goals/tailor.py

```diff
@@ -55,7 +55,7 @@ async def find_putative_targets(
         for dirname, filenames in group_by_dir(paths).items():
             putative_targets.append(
                 PutativeTarget.for_target_type(
-                    tgt_type, path=dirname, name=None, triggering_sources=sorted(filenames)
+                    tgt_type, path=dirname, name="lib", triggering_sources=sorted(filenames)
```

Comment:
Thoughts @tdyas and @sureshjoshi for what name is good for `cc_sources`?

(Reminder: this is temporary)

Reply:
Similar to python/shell? cc and cc-tests?
or cc-sources and cc-tests